### ISSUE TITLE
can I use it   simply

### ISSUE THREAD

**Issue by paulcst** [Castaglia/proftpd-mod_proxy#124]
Hello, developer
I have to say that this software is very difficult to configure and use. I have installed this software one day, even though I do not know how to use it, which is limited by my weak database accumulation. Is there a simpler way to configure usage?

**Comment by Castaglia** [comment]
No, there is not a simpler way to configure usage.  Proxying is a complex subject, with many things to tune and set, and thus the configuration needs to be able to handle this complexity.
